HD-Map/hdmap_shacl.ttl

```diff
@@ -0,0 +1,401 @@
+            sh:message "Validation of roadTypes failed!"@en ;
+            sh:name "roadTypes"@en ;
+            sh:order 0 ;
+            sh:path hdmap:roadTypes ],
+        [ skos:example "shoulder, curb, ..." ;
+            sh:datatype xsd:string ;
+            sh:description "Covered lane types, see ODR spec section 9.5.3."@en ;
+            sh:in ("biking" "border" "connectingRamp" "curb" "driving" "entry" "exit" "median" "none" "offRamp" "onRamp" "parking" "restricted" "shoulder" "slipLane" "stop" "walking") ;
+            sh:message "Validation of laneTypes failed!"@en ;
+            sh:name "laneTypes"@en ;
+            sh:order 1 ;
+            sh:path hdmap:laneTypes ],
+        [ skos:example "trees, street lamps, road patches..." ;
+            sh:datatype xsd:string ;
+            sh:description "Covered object classes, see ODR spec section 11"@en ;
+            sh:in ("barrier" "bike" "building" "bus" "car" "crosswalk" "gantry" "motorbike") ;
+            sh:message "Validation of levelOfDetail failed!"@en ;
+            sh:name "levelOfDetail"@en ;
+            sh:order 2 ;
+            sh:path hdmap:levelOfDetail ] ;
+    sh:targetClass hdmap:Content .
+
+hdmap:DataSourceShape a sh:NodeShape ;
+    sh:property [ skos:example "3DMS system, Trimble xyz, Riegl xyz" ;
+            sh:datatype xsd:string ;
+            sh:description "Main acquisition device"@en ;
+            sh:maxCount 1 ;
+            sh:message "Validation of measurementSystem failed!"@en ;
+            sh:name "measurementSystem"@en ;
+            sh:order 1 ;
+            sh:path hdmap:measurementSystem ],
+        [ skos:example "scanner, camera, osm network, aerial images, ..." ;
+            sh:datatype xsd:string ;
+            sh:description "Basic data for the creation of the map"@en ;
+            sh:message "Validation of usedDataSources failed!"@en ;
+            sh:name "usedDataSources"@en ;
+            sh:order 0 ;
+            sh:path hdmap:usedDataSources ] ;
+    sh:targetClass hdmap:DataSource .
+
+hdmap:FormatShape a sh:NodeShape ;
+    sh:property [ skos:example "1.5" ;
+            sh:datatype xsd:string ;
+            sh:description "Version of data format"@en ;
+            sh:maxCount 1 ;
+            sh:message "Validation of version failed!"@en ;
+            sh:name "version"@en ;
+            sh:order 1 ;
+            sh:path hdmap:version ],
+        [ skos:example "ASAM OpenDRIVE  Road5 Lanelet road2sim roadXML  Shape" ;
+            sh:datatype xsd:string ;
+            sh:description "Format type definition"@en ;
+            sh:in ("ASAM OpenDRIVE" "Lanelet" "Road5" "Shape" "road2sim" "roadXML") ;
+            sh:maxCount 1 ;
+            sh:message "Validation of type failed!"@en ;
+            sh:name "type"@en ;
+            sh:order 0 ;
+            sh:path hdmap:type ] ;
+    sh:targetClass hdmap:Format .
+
+hdmap:GeneralShape a sh:NodeShape ;
```

Comment:
There is also a "general:GeneralShape" defined by Mirco (https://github.com/GAIA-X4PLC-AAD/ontology-management-base/pull/27)
This one seems to be something different. Is "General" the right name for it?

As I understood Mirco all DataShapes (like HdMap) should include the `general:GeneralShape`. Did you forget it to include?

Reply:
please discuss in the meeting

Reply:
local general node was removed and generalShape was integrated. done!